dev/ci-build.sh

```diff
@@ -30,14 +30,25 @@ if [[ -z $CCL_ROOT ]]; then
  exit 1
 fi
 
+if [[ -z $1 ]]; then
+ echo SPARK_VER not defined, using default.
+else
+ SPARK_VER=$1
+fi
+
 echo === Building Environments ===
 echo JAVA_HOME=$JAVA_HOME
```

Comment:
It's better you put SPARK_VER in the beginning of building environments before JAVA_HOME, same below